specification/azsadmin/resource-manager/subscriptions/readme.md

```diff
@@ -64,6 +64,11 @@ directive:
     where:
       - $.paths["/subscriptions/{subscriptionId}/providers/Microsoft.Subscriptions.Admin/moveSubscriptions"].post.operationId
       - $.paths["/subscriptions/{subscriptionId}/providers/Microsoft.Subscriptions.Admin/validateMoveSubscriptions"].post.operationId
+
+  - where:
```

Comment:
why do we need this change here?

Reply:
I don't think that's needed. I am using readme.azsautogen.md instead of readme.md for subscription admin module. do you want me to take the changes from readme.azsautogen.md to readme.md here and use it for the subscription admin module generation? 

It seems this change was added by Vishnu to fix some generator issue
https://github.com/viananth/azure-rest-api-specs/commit/04297b97cf0dda43caae343edcb70d2ce626d95d

Reply:
Thank you Rakesh! I will revert that change if its not required.